features/user_views_artists.feature

```diff
@@ -5,3 +5,10 @@ Feature: User views artists listing
     When I open the app
     And I want to see a listing of the artists
     Then I should see 12 or fewer artists
+
+  Scenario: A user views an artist
+    Given the app has a few artists
+    When I open the app
+    And I want to see a listing of the artists
```

Comment:
Hmm...instead of doing
  ```
  When I open the app
  And I want to see a listing of the artists
  ```
We could do
  ```
  When I go to the artists page
  ```
The two separate steps feel a little clunky. What do you think @scooter-dangle?